src/App.jsx

```diff
@@ -1,107 +1,22 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+import { PostList } from './components/PostList';
+
+const prepearedPosts = postsFromServer.map(
+  post => ({
+    ...post,
+    user: usersFromServer.find(user => user.id === post.userId),
+    comments: commentsFromServer.filter(comment => comment.postId === post.id),
```

Comment:
Don`t forget about destructing feature:
```suggestion
    user: usersFromServer.find(({ id })=> id === post.userId),
    comments: commentsFromServer.filter(({ postId }) => postId === post.id),
```